components/layout_thread/lib.rs

```diff
@@ -1042,7 +1042,11 @@ impl LayoutThread {
             // Observe notifications about rendered frames if needed right before
             // sending the display list to WebRender in order to set time related
             // Progressive Web Metrics.
-            self.paint_time_metrics.maybe_observe_paint_time(self, epoch, &display_list);
+
+            // `maybe_observe_paint_time` expects a `&gfx_traits::DisplayList` instead
+            // of a `&gfx::display_list::DisplayList` so we need to help the typechecker
+            // here with `&*display_list`.
```

Comment:
I don't think this comment is necessary. We could also do `&display_list as &gfx_traits::DisplayList` if we wanted to make it more clear. I'm fine either way.

Reply:
Oh I didn't know you could do it that way, that is much clearer.